### PR TITLE
Add `logo_url` to Organization API response

### PR DESCRIPTION
**Pull request by tdooner**
This adds a new field to the Organization API response that contains a
URL to a logo for that organization. Currently this is only pulled from
the GitHub projects_list_url, which is not the be-all-end-all for a
couple reasons:

1- It's hard to tell if an organization has uploaded a logo or if github
generated them a default blocky logo.

2- Some organizations use a CSV file for their projects_list_url.

In the case of 1, we will currently use the placeholder logo. In the
case of 2, the logo_url will be null. This adds support for, but I have
not tested, a "logo_url" key/value in the brigade-information
organizations.json file also, for any brigades that wish to override
this with a custom image.

### CODE REVIEW

File: run_update.py [hunk 1]
```diff
@@ -1197,6 +1204,27 @@ def update_attendance(session, organization_name, attendance_dict):
     return existing_attendance
 
 
+def get_logo(org_info):
+    ''' get an organization's logo, looking first at Github (project lists url) '''
```
Comment (ondrae) [inline run_update.py]: It's first checking for an explicit `logo_url` right?

File: run_update.py [hunk 2]
```diff
@@ -44,6 +43,7 @@
 # TODO: use a Meetup client library with pagination
 MEETUP_API_URL = "https://api.meetup.com/2/events?status=past,upcoming&format=json&group_urlname={group_urlname}&key={key}&desc=true&page=200"
 MEETUP_COUNT_API_URL = "https://api.meetup.com/2/groups?group_urlname={group_urlname}&key={key}"
+GITHUB_USER_API_URL = 'https://api.github.com/users/{username}'
```
Comment (ondrae) [inline run_update.py]: I'm a little confused by the different use of `/user` and `/orgs` below. As long as you are clear on which to use when, then 👍

Comment (tdooner) [reply run_update.py]: It appears that using `/user/:id` works in both cases - whether the profile is a single user _or_ an organization - and at least one brigade has a single-user profile. But the general case for an organization. Good flag.

File: run_update.py (hunk 1 above)
Comment (tdooner) [reply run_update.py]: Good point.

**Comment by tdooner** [comment]
Having addressed Andrew's feedback, gonna merge this and watch it run this afternoon.

**Pull request merged**